api/metrics-api-jaxrs/src/main/webapp/WEB-INF/web.xml

```diff
@@ -24,6 +24,16 @@
 
   <display-name>Hawkular Metrics Rest interface</display-name>
 
+  <resource-env-ref>
+    <resource-env-ref-name>container/hawkular-alerts</resource-env-ref-name>
+    <lookup-name>java:jboss/infinispan/container/hawkular-alerts</lookup-name>
+  </resource-env-ref>
+
+  <resource-env-ref>
+    <resource-env-ref-name>cache/locks</resource-env-ref-name>
+    <lookup-name>java:jboss/infinispan/cache/hawkular-alerts/locks</lookup-name>
+  </resource-env-ref>
+
```

Comment:
So, prior to this change Metrics nodes did not require clustering, right?  Although, non-clustered nodes meant that Alerting would likely have had to be disabled.   Now Metrics requires clustering and ISPN for schame handling, right?  This seems OK to me given the Alerting needs and the overall strategy to have clustered nodes, but I just wanted to confirm that this was basically true.